endoreg_db/views/video/correction.py

```
@@ -542,14 +543,18 @@ def post(self, request, pk):
                 frames_to_remove=frames_to_remove,
-                output_video=str(output_path)
+                output_video=output_path
             )
             
             processing_time = time.time() - start_time
             
             if success:
                 # Update video record
-                video.anonymized_file = f"anonym_videos/{video.uuid}_cleaned.mp4"
+                processed_field = video.processed_file
+                expected_cleaned_path = Path(processed_field.path)
```

Ruff (0.14.1)

Local variable `expected_cleaned_path` is assigned to but never used

Remove assignment to unused variable `expected_cleaned_path`

(F841)